### PR TITLE
Fixed the broken codeql action

### PR DESCRIPTION
**Pull request by naveensrinivasan**

### CODE REVIEW

File: .github/workflows/codeql-analysis.yml [hunk 1]
```diff
@@ -13,7 +13,6 @@ name: "CodeQL"
 
 on:
   push:
-    branches: [ main, * ]
```
Comment (laurentsimon) [inline .github/workflows/codeql-analysis.yml:16]: is this going to run on push to all branches?

File: .github/workflows/codeql-analysis.yml [hunk 2]
```diff
@@ -32,9 +31,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        language: [  ]
-        # CodeQL supports [ 'cpp', 'csharp', 'go', 'java', 'javascript', 'python', 'ruby' ]
-        # Learn more about CodeQL language support at https://git.io/codeql-language-support
+        language: [ 'javascript' ]
```
Comment (laurentsimon) [inline .github/workflows/codeql-analysis.yml:34]: why not golang?

File: .github/workflows/codeql-analysis.yml (hunk 1 above)
Comment (naveensrinivasan) [reply .github/workflows/codeql-analysis.yml:16]: Yes! The previous syntax was wrong.

File: .github/workflows/codeql-analysis.yml (hunk 2 above)
Comment (naveensrinivasan) [reply .github/workflows/codeql-analysis.yml:34]: There isn't a golang file in this repository.

Comment (laurentsimon) [reply .github/workflows/codeql-analysis.yml:34]: oops, sorry... I though we were on scorecard repo :-)
If there's bash script and dockerfile support, we could enable it

Comment (laurentsimon) [reply .github/workflows/codeql-analysis.yml:34]: actually I'm curious: what does CodeQl check for if we don't tell it which languages to support? Does it detect it automatically? We have only bash script and docker file in this repo.

Comment (naveensrinivasan) [reply .github/workflows/codeql-analysis.yml:34]: It checks for the GitHub actions. We need a js script file to check for vulnerabilities.

Comment (laurentsimon) [reply .github/workflows/codeql-analysis.yml:34]: OK.

**Review by laurentsimon — APPROVED** [review]
Thanks @naveensrinivasan and sorry for these endless questions :-)

**Comment by naveensrinivasan** [comment]
> Thanks @naveensrinivasan and sorry for these endless questions :-)

👍

**Pull request merged**